src/Controllers/ToolbarController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Viewer\Controllers;
+
+use Nyholm\Psr7\Response;
```

Comment:
Can we use response factory instead of relying on concrete response implementation?

Reply:
Yes, sure